### ISSUE TITLE
a11y: Color Contrast - User Retention Settings

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13737]
Insufficient color contrast for standard text

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF 
Ratio: 3.16:1 
Main Content
- Users: 
- Cancel (on default state) 
- Inactivity period, User retention process schedule, Default last login (ms) (on hover and focus)

Foreground Text Color: #FFFFFF
Background Text Color: #3D98D3
Ratio: 3.16:1 
Main Content
- Save button (on default state) 

Foreground Text Color: #BEC1D2
Background Text Color: #FFFFFF 
Ratio: 1.78:1 
Main Content
- Unix timestamp

Foreground Text Color:  #FFFFFF 
Background Text Color: #297DB4
Ratio: 4.48:1 
Main Content
- Cancel and Save button (on focus ) 

"

"Toggle circle color: #FFFFFF
Inside background color: #DCDEE7
Ratio: 1.3:1
Main Content
- Run the user retention process...

Inside background color:#DCDEE7
outside background color: #FFFFFF
Ratio: 1.3:1
Main Content
- Run the user retention process…"

## Expected result

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Fails VPAT. No notes. Original feedback:

> The toggle form control have insufficient contrast with their background, making it difficult for people with low vision and color blindness to distinguish them.

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
> 
> Instances:
> Foreground Text Color: #BEC1D2
> Background Text Color: #FFFFFF 
> Ratio: 1.78:1 
> Main Content
> - Unix timestamp
> 
> Foreground color: #2F68DF
> Background color: #EDF2FC
> Contrast ratio: 4.4:1
> - Cancel (On hover on focus )
> 
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text."

Issue 203
Issue 204
